asm2vec/binary_to_asm.py

```diff
@@ -98,14 +102,15 @@ def bin_to_asm(filename: Path, output_path: Path, asm_minlen: int) -> int:
     return count
 
 
-def convert_to_asm(input_path, output_path, minlen_upper: int, minlen_lower: int) -> list:
+def convert_to_asm(input_path, output_path, minlen_upper: int, minlen_lower: int, magic_bytes='cffaedfe') -> list:
     """ Extracts assembly functions from malware files and saves them
     into separate folder per binary
     :param input_path: the path to the malware binaries
     :param output_path: the path for the assembly functions to be extracted
     :param minlen_upper: The minimum number of assembly functions needed for disassembling
     :param minlen_lower: If disassembling not possible with with minlen_upper, lower the minimum number
     of assembly functions to minlen_lower
+    :param magic_bytes for the specific OS/type of binary
```

Comment:
What is the default and to which OS does it link?

Reply:
The default values correspond to MacOS - added a list of valid magic bytes